src/scripts/install_machine.py

```diff
@@ -39,6 +39,15 @@ def get_graphics_capabilies(connection):
         for value in graphics.find('enum').findall('value'):
             consoles.append(value.text)
 
+    # HACK: Ignore spice on RHEL 8; https://issues.redhat.com/browse/RHEL-18058
+    try:
+        with open("/etc/os-release") as f:
+            if "platform:el8" in f.read():
+                logging.debug("get_graphics_capabilies: ignoring spice on RHEL 8")
+                consoles.remove('spice')
+    except FileNotFoundError:
+        pass  # not RHEL then
```

Comment:
So hmm spice is more then just the console node. Who adds the other things like spice-serial? OR does this happen before virt-install is done?

I do spot a test below for any occurrence of spice in the XML but I don't know enough details if that is 100% correct.

Also this would be a "breaking" change in RHEL8, is that allowed/normal?